newIDE/app/src/MainFrame/index.js

```diff
@@ -2076,5 +2070,5 @@ class MainFrame extends React.Component<Props, State> {
     );
   }
 }
-
+MainFrame.contextType = PreferencesContext;
```

Comment:
This will fail the flow test :( I don't know how to use flow with `contextType`, I've checked their docs and didn't find anything. Though we don't really need to use `contextType` here, we could've stored stuff into `recentFiles` during the rendering of `SceneEditor`, but there was a warning by react which basically said don't update anything while rendering.